mmdet3d/datasets/pipelines/dbsampler.py

```diff
@@ -245,7 +254,9 @@ def sample_all(self, gt_bboxes, gt_labels, img=None):
                     self.data_root,
                     info['path']) if self.data_root else info['path']
                 s_points = np.fromfile(
-                    file_path, dtype=np.float32).reshape([-1, 4])
+                    file_path,
```

Comment:
Support to use `fileclient`. Therefore, we may consider using `LoadPointsFromFile` here.